pubspec.yaml

```
@@ -40,10 +40,59 @@ dev_dependencies:
   slang_build_runner: ^3.32.0
   freezed: ^2.5.7
   json_serializable: ^6.8.0
+  rps: ^0.9.1
+
+dependency_overrides:
+  # TODO rps override
+  meta: ^1.16.0 
```

yamllint

[error] trailing spaces

(trailing-spaces)